components/PackageSidebar.tsx

```diff
@@ -70,6 +72,14 @@ export default function PackageSidebar({
     router.push(`/packages/${packageName}/versions/${selectedVersion}`);
   }
 
+  useEffect(() => {
```

Comment:
You could extract this as a hook which you can test like eg `useHasCollaborator` which returns a boolean. That way you can test the hook in isolation, and have a single-purpose function

Reply:
needs unit test setup. creating a ticket for it